tasks/kernel_matrix_testing/stacks.py

```diff
@@ -293,7 +293,7 @@ def pause_stack(stack=None):
     conn.close()
 
 
-def resume_stack(stack=None):
+def resume_stack(_, stack=None):
```

Comment:
Why the added unused argument?

Reply:
Oh good catch. Added it in the wrong place, Should be in the `resume_stack` function which is a task.